### PR TITLE
Only create previews on branches (not forks)

### PR DESCRIPTION
**Pull request by sdepold**

### CODE REVIEW

File: .github/workflows/draft.yml [hunk 1]
```diff
@@ -34,21 +35,20 @@ jobs:
           netlify deploy --dir build --message '${{ github.event.commits[0].message }}' > log.tmp.txt 2>&1
           cat log.tmp.txt | grep -E 'Logs|Website Draft URL' > log.txt
       - name: Read deployment log
+        if: '! github.event.pull_request.head.repo.fork'
```
Comment (WikiRik) [inline .github/workflows/draft.yml:38]: Let's make them dependent on eachother using `needs:` like we do in the `core` CI.

Comment (sdepold) [reply .github/workflows/draft.yml:38]: You mean a `needs` in a step? I don't think that is possible

Comment (WikiRik) [reply .github/workflows/draft.yml:38]: Oh wait, these are not separate steps. Nevermind then. I was just thinking of a way to only do the logic for the `if` in one place so we don't forget the others when we add extra conditions

**Review by ephys — APPROVED** [review]
This looks good to me 
image

File: .github/workflows/draft.yml (hunk 1 above)
Comment (sdepold) [reply .github/workflows/draft.yml:38]: Completely agreed but I couldn't find a less ugly way :)

**Comment by sdepold** [comment]
@WikiRik I think you will have to approve one more time

**Review by WikiRik — APPROVED** [review]

**Comment by WikiRik** [comment]
We had to resolve the conversation, but that's all done now. Thanks for this PR!

**Pull request merged**